jobserver/models.py

```diff
@@ -428,6 +477,31 @@ def manifest(self):
         return json.loads(self.file_path("metadata/manifest.json").read_text())
 
 
+class ResearcherRegistration(models.Model):
+    """
+    The Registration of a Researcher for a Project
+
+    When registering a new Project 0-N researchers might be named as part of
+    the Project.  Instead of requiring they sign up to the service during the
+    registration process this model holds their details, and can be attached to
+    a User instance if the Project proceeds past registration.
+    """
+
+    user = models.ForeignKey(
+        "User",
+        null=True,
+        on_delete=models.SET_NULL,
+        related_name="researcher_registrations",
+    )
+
+    name = models.TextField()
+    passed_researcher_training_at = models.DateTimeField()
```

Comment:
Could this just be a DateField?

Reply:
It could, but I always opted to use DateTimeFields over DateFields because they so often become the latter.